### DISCUSSION TITLE
Email converted to ticket - attachment fails

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16668] [General] [answered]
When email with attachment is converted to ticket, this attachment is not set in this ticket. Is it my miss-configuration or design?

**Comment by mariuszkrzaczkowski** [comment]
What version of CRM do you have?

**Comment by s2b-git** [comment, marked as answer]
Latest stable, but fixed. Some cofig entry which now I dont remember. Sorry for noise ;-)
